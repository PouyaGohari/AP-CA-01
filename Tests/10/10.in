3
a b c
a:0 YES|1 NO|2 IF_NECESSARY