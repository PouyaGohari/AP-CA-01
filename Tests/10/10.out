Time slot 0:
YES: a 
NO: 
IF_NECESSARY: b c 
###
Time slot 2:
YES: 
NO: 
IF_NECESSARY: a b c 
###
Time slot 1:
YES: 
NO: a 
IF_NECESSARY: b c 
###
